stdlib/gc.mli

```diff
@@ -300,7 +304,9 @@ external full_major : unit -> unit = "caml_gc_full_major"
    unreachable blocks. *)
 
 external compact : unit -> unit = "caml_gc_compaction"
-(** Perform a full major collection and compact the heap.  Note that heap
+(** Note: As of OCaml 5.0, compaction is not implemented, and calling this
+   function will only trigger a major collection.
+   Perform a full major collection and compact the heap.  Note that heap
```

Comment:
I was going to suggest that this should presumably be equivalent to `Gc.full_major` ... but in fact it looks like there's a bug in the implementation, as the C code appears only to do a major GC slice!